### ISSUE TITLE
Request for .pbtxt file

### ISSUE THREAD

**Issue by craston** [AIZOOTech/FaceMaskDetection#52]
Hi,  Great work! Is it possible to get the ..pbtxt file

**Comment by AIZOOTech** [comment]
@craston You can try to convert pb file to pbtxt. I train the model with Keras. It can convert to pbtxt theoretically.

**Comment by hunglv213** [comment]
Hi, do you follow the baseline training code from any github?

**Comment by AIZOOTech** [comment]
@hunglv213 No, actually we are the baseline for ohters. You can read this paper [RETINAMASK: A FACE MASK DETECTOR](https://arxiv.org/pdf/2005.03950.pdf)
